app/model/Upload.scala

```diff
@@ -0,0 +1,16 @@
+package model
+
+import org.cvogt.play.json.Jsonx
+import play.api.libs.json._
+
+case class UploadPolicy(bucket: String, key: String, region: String, credentials: UploadCredentials)
+case class UploadCredentials(temporaryAccessKey: String, temporarySecretKey: String, sessionToken: String)
```

Comment:
minor: could it be `temporaryAccessId` to keep inline with the way AWS names it?